### DISCUSSION TITLE
rEN A IS LOADING SLOWLY

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#1019] [(D) Developer]
the arena battles are loading veeeeeeeeeeeeeeeeeeeeeeeeeeeeeery slow. Even though i didnt use any arena hacks.
BTW:I s there a way to bring you account back to normal?

**Comment by hostedposted** [comment]
did you use the skip tutorial hack?

**Reply by ghost** [reply]
no its working better now. Np

**Reply by hostedposted** [reply]
okay
